dataprofiler/labelers/column_name_model.py

```diff
@@ -109,8 +126,15 @@ def _validate_parameters(self, parameters):
                         param
                     )
                 )
+            elif param == "positive_threshold_config" and (
+                value is None or not isinstance(value, int)
+            ):
+                errors.append(
+                    "`{}` is a required parameter that must be a boolean.".format(param)
+                )
```

Comment:
moving this back to `column_name_model.py` to allow for `show_confidences` in the `ColumnNameDataLabeler`